src/SecurityInsights/examples/Get-AzSentinelMetadata.md

```diff
@@ -0,0 +1,20 @@
+### Example 1: {{ Add title here }}
+```powershell
+ {{ Add code here }}
+```
+```output
+{{ Add output here }}
```

Comment:
Could you fill all placeholders in examples?

Reply:
Fixed